server/src/test/java/io/spine/server/aggregate/AggregateRepositoryTest.java

```diff
@@ -499,7 +499,7 @@ void afterCommand() {
                     .newBuilder()
                     .setProjectId(id)
                     .build();
-            BlackBoxBoundedContext.newInstance()
+            BlackBoxBoundedContext.singletenant()
```

Comment:
Can we please have `newInstance()` back, meaning "single-tenant context" by default.